src/components/analytics.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component, Fragment } from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router'
+
+class Analytics extends Component {
+
+  componentDidUpdate(props) {
+    this.trackHashChange(props)
+  }
+
+  trackHashChange({ location, history }) {
+    const gaTrackingId = process.env.GA_TRACKING_ID || 'UA-106384880-2'
```

Comment:
Is this default value UA-106384880-2 needed ? is there a risk it could cause data from local testing and staging to get tracked/pollute the mainnet data set ?

Reply:
@franckc unfortunately gtags.js requires the ID to be passed into the `'config'` call. However, the `gtag` global var is undefined on localhost because the script is prevented from running on `dev.html`, so it won't track local pageviews/events.